src/substitutions.ts

```diff
@@ -0,0 +1,317 @@
+import { Logging } from "matrix-appservice-bridge";
+import * as emoji from "node-emoji";
+import { Main } from "./Main";
+import { ISlackFile } from "./BaseSlackHandler";
+
+const log = Logging.get("substitutions");
+
+export function onMissingEmoji(name) {
```

Comment:
This is not an event handler, it is a request. I suggest `getFallbackForMissingEmoji`